Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.7-bullseye
```

Comment:
Do we need the ability to run the algorithm on the GPU? If yes, then you should look at the [NVIDIA Container Toolkit](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/overview.html)

Reply:
The original script used the GPU for evaluation, however @vnmsklnk modified it to use CPU if the GPU is unavailable